px/main.py

```diff
@@ -1043,6 +1046,7 @@ def restore_stdout():
             State.debug.stdout = State.stdout
 
     def attach_console():
```

Comment:
In this case, we should probably comment out the origianl call to `attach_console()`